README.md

```diff
@@ -163,9 +165,11 @@ editing bindings. So, for example:
 * All your standard horizontal motion, selection, insertion, and
   deletion commands work as usual.
 * To delete your current input, just use `C-a C-k`.
-* To go up a directory, use `M-DEL` to delete it in the minibuffer.
+* To edit the input use `M-DEL` or `C-M-DEL` to go up a directory.
```

Comment:
Maybe explain the difference between these two commands?

Reply:
Done